src/MetricsUploader/ScopedMetricTest.cpp

```diff
@@ -38,8 +38,8 @@ TEST(ScopedMetric, Constructor) {
 
   MockUploader uploader{};
 
-  EXPECT_CALL(uploader, SendLogEvent(OrbitLogEvent::ORBIT_MAIN_WINDOW_OPEN, _,
-                                     OrbitLogEvent_StatusCode_SUCCESS))
+  EXPECT_CALL(uploader,
+              SendLogEvent(OrbitLogEvent::ORBIT_MAIN_WINDOW_OPEN, _, OrbitLogEvent::SUCCESS))
```

Comment:
completely unrelated, but shouldn't we get rid of the "_"?

Reply:
Yes, but its not possible in ORBIT_MAIN_WINDOW_OPEN and the _ after that is unrelated (related to EXPECT_CALL)